app/src/main/java/io/pumpkinz/pumpkinreader/MainActivity.java

```diff
@@ -8,9 +8,7 @@
 import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
 
-
 public class MainActivity extends AppCompatActivity {
-
```

Comment:
I prefer to have blank line after class declaration & before class closing bracket.

Also it's better to have double blank lines between import and class declaration. It's just easier for the eyes.